cluster/examples/kubernetes/nfs/storageclass.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: storage.k8s.io/v1
+kind: StorageClass
+metadata:
+  name: rook-nfs
+provisioner: nfs.rook.io/nfs
+parameters:
+  # Name of the nfs-daemon
```

Comment:
maybe this comment should be more in the context of the Rook experience and use less of the NFS terminology.  Something like `Name of the networkfilesystem.nfs.rook.io instance`